index.js

```diff
@@ -121,6 +121,10 @@ module.exports = function (options) {
     throw new Error('No correct handler set. It\'s needed to actually process a Micropub request.');
   }
 
+  if (!options.syndication || options.syndication.constructor !== Array) {
```

Comment:
First – thanks for the PR! Lovely to get contributions from others! I will go through and add some feedback on the different pieces now.

If the data is not an array then I think it would be better for an error to be thrown, so I would write this as the following instead:

```javascript
options.syndication = options.syndication || [];
if (!Array.isArray(options.syndication)) {
  throw new Error('Incorrect syndication options defined, expected an array');
}
```

And perhaps even do `options.syndication = [].concat(options.syndication || []);` to allow setting a single value without wrapping it in an array.

*But* – what I in this case would even more do is to make the handling of query data into a callback in a similar way as `tokenReference` and `handler` are. The reason for that is that a Micropub endpoint can be shared between many different users which all have their individual syndication preferences (or [mp-config](http://indiewebcamp.com/mp-config) for that matter) and then it's hard to define the values up front on creation.

So I think, that to keep this endpoint lean and simple I would opt for an optional `queryHandler` option instead, that's a callback that receives maybe two params, `(queryType, req)` (similar to the `(micropubDocument, req)` of `handler`).

That callback could then return either `false` if it doesn't know how to handle the current request or an object that could be transformed to the response data, something like:

```javascript
{
  'syndicate-to': ['http://twitter.com/voxpelli']
}
```

How does that sound?